apycula/tiled_fuzzer.py

```diff
@@ -28,8 +28,8 @@
 if not gowinhome:
     raise Exception("GOWINHOME not set")
 
-# device = os.getenv("DEVICE")
-device = sys.argv[1]
+device = os.getenv("DEVICE")
+# device = sys.argv[1]
```

Comment:
Reminder that I'm an idiot.
I went through the trouble of `git add -p` and then just muscle memoried `git commit -am`